packages/client/hmi-client/src/components/workflow/ops/intervention-policy/tera-intervention-card.vue

```diff
@@ -29,6 +29,7 @@
 				<tera-input-number
 					auto-width
 					:model-value="intervention.staticInterventions[0].timestep"
+					:invalidate-negative="true"
```

Comment:
we can just do `invalidate-negative` if the prop is always true (see auto-width)